api/api_sources/sources/server/modules/observation/observation.route.ts

```diff
@@ -35,6 +36,15 @@ export class ObservationRouteController extends ResourceRouteController<Observat
     public async search(req: any, data?: any): Promise<[number, any]> {
         return [200, await this.dataController.search(req.query.keyword)];
     }
+
+    @Get({
+        path: '/export',
+        secure: true,
+        middleware: () => [ inspectAppAdminRoute() ]
```

Comment:
inspectAppAdminRoute() is only allow admin and inspect app admin, please get confirmation from Mike. I believe this route will be either adminOnly or writer.